tests/examples/pacer/nda/ca2_5.json

```diff
@@ -15,7 +15,8 @@
           "pacer_case_id": "56342",
           "pacer_doc_id": "00208962225",
           "pacer_magic_num": "8d96e8f0a9737656",
-          "pacer_seq_no": null
+          "pacer_seq_no": null,
+          "short_description": "Appendix FILED"
```

Comment:
And one more time?

Reply:
In this case the description is the same:

![Screenshot 2023-04-21 at 16 44 45](https://user-images.githubusercontent.com/486004/233746552-244bc145-b2ef-4c66-83a8-0c3786cc965e.png)

![Screenshot 2023-04-21 at 16 44 59](https://user-images.githubusercontent.com/486004/233746551-e2c6be14-8762-4f61-b306-124b8ad1c288.png)

Reply:
Cool.